api/swagger.yaml

```diff
@@ -9238,6 +9238,19 @@ paths:
           in: "query"
           description: "The tag to associate with the image on the registry."
           type: "string"
+        - name: "platform"
+          type: "string"
+          in: "query"
+          description: |
+            JSON encoded OCI Platform to select a platform-specific manifest
+            matching this platform to be pushed.
+            If not provided, full multi-platform image will be pushed.
+            Note: Selection is only supported with the containerd image store.
+              With graphdrivers image store, it will only check that the actual
+              platform of the image matches the provided platform and error out
+              otherwise.
```

Comment:
With containerd, wouldn't we also produce an error if I specify `--platform <something not present locally>` ? Because if that's the case we may not have to document the distinction between the two.

(basically, it produces an error when the given platform is not present?)

Reply:
Yes, it will also error out.
The distinciton I wanted to make here is that with containerd we will actually pick a specific platform to push (because one image can now have multiple platform available). While with graphdrivers a single image is always single platform (so there's nothing to "choose" from)/

Reply:
I think @thaJeztah's suggestion is excellent!